src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -252,4 +252,13 @@
   <data name="InvalidConstraint" xml:space="preserve">
     <value>Constraint is not supported for this type and will be ignored.</value>
   </data>
+  <data name="NonAppJsonTypeNotSupported" xml:space="preserve">
+    <value>Non application json MIME type has limited support</value>
+  </data>
+  <data name="SupportedSchemesWarningMessage" xml:space="preserve">
+    <value>Non-HTTPS schemes have limited Autorest support</value>
```

Comment:
o_O what are we saying about HTTP? We support HTTP, right?

Again, maybe be clearer `Support for schemes other than {"http","https" } have limited support `